src/test/scala/com/campudus/tableaux/IdentifierTest.scala

```diff
@@ -51,6 +51,27 @@ class IdentifierTest extends TableauxTestBase {
   }
 
   @Test
+  def retrieveConcatColumn(implicit c: TestContext): Unit = okTest {
+    val createStringColumnJson = Json.obj("columns" -> Json.arr(Json.obj("kind" -> "text", "name" -> "Test Column 3")))
+
+    for {
+      _ <- setupDefaultTable()
+
+      // create a third column
+      _ <- sendRequest("POST", s"/tables/1/columns", createStringColumnJson)
+
+      // make the first and the last an identifier column
+      _ <- sendRequest("POST", "/tables/1/columns/1", Json.obj("identifier" -> true))
+      _ <- sendRequest("POST", "/tables/1/columns/3", Json.obj("identifier" -> true))
```

Comment:
General question - is it possible to create a column and set the identifier to true at creation? Or do we always need to update columns?